src/m.camera.js

```diff
@@ -105,9 +105,12 @@ please.CameraNode = function () {
     ANI("near", 0.1);
     ANI("far", 100.0);
 
+    if (please.gl.ctx === null)
+        this.location_z = 100.0;
+
     this.mark_dirty();
     this.projection_matrix = mat4.create();
-    this.__projection_mode = "perspective";
+    this.__projection_mode = please.gl.ctx === null ? "orthographic" : "perspective";
```

Comment:
Another place where the property should never change, so a conditional statement following the original definition which sets it to orthographic and then calls Object.freeze(this.__projection_mode);